src/util.ts

```diff
@@ -249,3 +249,13 @@ function lastDotCmp(s: string): string {
 	const cmps = s.split('.');
 	return cmps[cmps.length - 1];
 }
+
+/**
+ * Strips file part (if any) from container name (last component of container path)
+ * For example TS may return the following name: /node_modules/vscode-jsonrpc/lib/cancellation.
+ * We consider that if name contains path separtor then container name is empty
+ * @param containerName
+ */
+function stripFileInfo(containerName: string): string {
```

Comment:
When we looking at symbol in project it's container name may look like `/node_modules/path-to/file` but when we are searching this symbol in it's own repo container name is empty (top-level symbol). This making two symbols not match when we are trying to resolve cross-repo symbol location. I think we need to process container name as returned by TS the following way:
- extract last component (after last dot if any)
- check if there are path separators, if there are - I assume we are looking at top-level element defined in some file so container name is empty string